apps/web/src/app/[lang]/app/[type]/crm/actions/merchant.ts

```diff
@@ -46,3 +49,101 @@ export async function getCrmServiceMerchantsDetailById(
     return structuredError(error);
   }
 }
+
+export async function getMerchantsByIdSubMerchants(
```

Comment:
Can we fix the function name?
```suggestion
export async function getSubMerchantsByMerchantId(
```